exporter/elasticsearchexporter/factory.go

```diff
@@ -60,6 +65,14 @@ func createDefaultConfig() component.Config {
 				http.StatusGatewayTimeout,
 			},
 		},
+		BatcherConfig: exporterbatcher.Config{
+			Enabled:      true,
+			FlushTimeout: 30 * time.Second,
+			MinSizeConfig: exporterbatcher.MinSizeConfig{
+				MinSizeItems: 125,
```

Comment:
Agree that it is very conservative. Let's find a better number.

I got it from logstash default `pipeline.batch.size` though I'm not sure if it has the same meaning: https://www.elastic.co/guide/en/logstash/current/logstash-settings-file.html

Reply:
Until we have a proper bytes-based threshold, I suggest we keep the `flush.bytes` config and default, and use that and an assumed average record size to calculate MinSizeItems. Say 1-2KB. Then this PR can be less breaking.

Reply:
The way I understand your comment is that, we 
1. do not deprecate `flush.bytes`
2. keep the `flush.bytes` `5000000` default
3. remove `batcher.min_size_items` default
4. if `batcher.min_size_items` is zero, calculate it from `flush.bytes/1KB`, otherwise, use `batcher.min_size_items` and ignore `flush.bytes`.

Also, do you reckon a good max_size_items default? Let's say 50MB is too much, and we work it backwards to get max_size_items=~50000?

Reply:
> do not deprecate flush.bytes

I would still deprecate it, just don't make setting it lead to an error.

> keep the flush.bytes 5000000 default

:+1: 

> remove batcher.min_size_items default

:+1: 

> if batcher.min_size_items is zero, calculate it from flush.bytes/1KB, otherwise, use batcher.min_size_items and ignore flush.bytes.

:+1: 

> Also, do you reckon a good max_size_items default? Let's say 50MB is too much, and we work it backwards to get max_size_items=~50000?

That seems too high to me. Maybe 10000.

Reply:
> That seems too high to me. Maybe 10000.

I realize setting a default max_size_items removes the possibility to set it back to 0, but this shouldn't present a problem.